app/components/show-dual-point-list.jsx

```diff
@@ -0,0 +1,117 @@
+// https://github.com/EnCiv/civil-pursuit/issues/57
+
+'use strict'
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import PointGroup from './point-group'
+import { ModifierButton } from './button'
+
+export default function ShowDualPointList({
+  className,
+  leftPoints = [],
+  leftHeader,
+  rightPoints = [],
+  rightHeader,
+  vState,
+  ...otherProps
+}) {
+  const classes = useStylesFromThemeFunction()
+  const [isExpanded, setIsExpanded] = useState(vState === 'expanded')
+  const toggleExpandCollapse = () => setIsExpanded(!isExpanded)
+
+  return (
+    <div className={cx(classes.sharedBorderStyle, className)} {...otherProps}>
+      <div className={classes.contentContainer}>
+        <div className={classes.header}>
+          <div className={classes.leftHeader}>{leftHeader}</div>
+          <div className={classes.rightHeader}>{rightHeader}</div>
+        </div>
+        <div className={classes.pointGrid}>
+          {leftPoints.map((point, index) => (
```

Comment:
It will be rare that there are an equal number of left and right points.  

- [ ] modify this component to deal with unequal numbers of left and right points
- [ ] create test case with differing number of left and right points
- [ ] create test case when there are no points on one side